content/running_bookwyrm/install-prod-dockerless.md

```diff
@@ -82,7 +82,7 @@ GRANT ALL PRIVILEGES ON DATABASE bookwyrm TO bookwyrm;
     - Make the system bookwyrm user:
         `useradd bookwyrm -r`
     - Change the owner of your install directory to bookwyrm:
-        `chown -R bookwyrm:bookwyrm /opt/bookwyrm`
+        `chown -R /opt/bookwyrm bookwyrm:bookwyrm`
```

Comment:
I think this is a mistake, the `chown` command work like `chown -R user:group directory`.

Reply:
corrected.